newIDE/app/src/AssetStore/AssetStoreSearchFilter.js

```diff
@@ -261,6 +261,155 @@ export class ColorAssetStoreSearchFilter
   }
 }
 
+const toAssetStoreType = (type: string) => {
+  return type === 'Sprite' ? 'sprite' : type;
+};
+
+// Thematic tags are noise for asset swapping as changing the theme may be what
+// users are looking for.
+const ignoredThematicTags = new Set([
```

Comment:
I wonder if "thematic tags" could be a category communicated by the backend in the future?